src/SchemaGameDB.js

```diff
@@ -0,0 +1,35 @@
+const Sequelize = require('sequelize');
+
+function generateSchemaGame(instanceSequalize) {
```

Comment:
it should be "generateGameSchema" or "buildGameSchema" and"sequalizeInstance"

Reply:
Changed the name of the file and variables